website/websitecore/static/websitecore/css/landing_page.css

```diff
@@ -0,0 +1,522 @@
+    background: white;
+    box-shadow: 10px 10px 5px rgba(0, 0, 0, 0.3);
+    position: absolute;
+    font-size: 9px;
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    margin-left: -838px;
+}
+
+.navbar ul a li:hover+.sub-navbar-2 .trak-2-grid img {
+    height: 70px;
+}
+
+.navbar ul a li:hover+.sub-navbar-2 .trak-2-grid {
+    gap: 20px;
+}
+
+.navbar ul a li:hover+.sub-navbar-2 h4 {
+    font-size: 16px;
+}
+
+.tresor {
+    background-image: url(../img/Front.jpg) ;
+    background-repeat: no-repeat;
+    padding: 10px 0;
+    height: 600px;
+    background-size: cover;
+}
+
+.services h3 {
+    padding-left: 200px;
+    font-size: 56px;
+    z-index: 1;
+}
+
+.service-content:hover {
+    background: none;
+}
+
+.service-grid {
+    display: grid;
+    grid-template-columns: repeat(4, 1fr);
+    grid-template-rows: 500px;
+}
+
+.column-content {
+    background-color: rgba(0, 47, 255, 0.795);
+    height: 100%;
+    opacity: 0.9;
+}
+
+.column-content:hover {
+    background: none;
+}
+
+.column p {
+    font-size: 30px;
+    margin-top: -1px;
+}
+
+.column p {
+    padding-top: 400px;
+    text-align: center;
+    color: white;
+    font-family: 'Poppins', sans-serif;
+}
+
+.column ul {
+    display: none;
+    width: 100%;
+    animation: mymove 500ms;
+}
+
+.column:hover ul {
+    display: block;
+    position: absolute;
+    color: white;
+    font-size: 25px;
+    line-height: 40px;
+    margin-top: 150px;
+    font-family: 'Poppins', sans-serif;
+    height: 100%;
+    overflow: hidden;
+}
+
+.column:hover {
+    animation: mymove;
+}
+
+@keyframes mymove {
+    0% {
+        top: 1200px;
+        position: relative;
+    }
+
+    5% {
+        top: 1100px;
+        position: absolute;
+    }
+
+    100% {
+        top: 1000px;
+        position: absolute;
+    }
+}
+
+.column:hover p {
+    display: none;
+}
+
+.about-us {
+    text-align: center;
+    font-family: 'Poppins', sans-serif;
+}
+
+.about-content {
+    padding: 123px 300px;
+    background-color: white;
+    opacity: 0.8;
+}
+
+.background-img {
+    background-image: url(../img/Rectangle\ 10.png);
+    height: 500PX;
+    background-attachment: fixed;
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: cover;
+}
+
+.case-study h3 {
+    padding-left: 200px;
+    font-size: 56px;
+    z-index: 1;
+}
+
+.study-grid {
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    grid-template-rows: 500px;
+}
+
+.block h4 {
+    padding-top: 410px;
+    text-align: center;
+    color: white;
+    font-size: 30px;
+    font-family: 'Poppins', sans-serif;
+}
+
+.block p {
+    display: none;
+    color: white;
+    background: none;
+}
+
+.block:hover p {
+    display: block;
+    position: absolute;
+    color: white;
+    width: 355px;
+    margin-left: 30px;
+    margin-top: 340px;
+    font-size: 20px;
+    animation: moveup 300ms;
+    font-family: 'Source Sans Pro', sans-serif;
+}
+
+@keyframes moveup {
```

Comment:
You need to review the logic you are using for this animation. 
This logic is going to make responsive too difficult.